src/main/kotlin/org/seekerwing/aws/sqsconsumer/configuration/MessageProviderConfiguration.kt

```diff
@@ -0,0 +1,19 @@
+package org.seekerwing.aws.sqsconsumer.configuration
+
+import org.seekerwing.aws.sqsconsumer.model.Queue
+
+/**
+ * [MessageProviderConfiguration] provides configuration options for
+ * [org.seekerwing.aws.sqsconsumer.messageprovider.MessageProvider].
+ * [queue] defines [Queue] from which the [org.seekerwing.aws.sqsconsumer.messageprovider.MessageProvider] will provide
+ * messages to the [org.seekerwing.aws.sqsconsumer.MessageProcessor].
+ * [messageFetcherConfiguration] defines [MessageProviderConfiguration] to be provide configuration valued for
+ * [org.seekerwing.aws.sqsconsumer.sqs.MessageFetcherKt.fetchMessage] to fetch messages from the [Queue].
+ * [parallelism] defines the number of coroutines/parallel executors what will be spun up to
```

Comment:
that will b spun from*

Reply:
i think the current phrasing is correct :|

Reply:
apologies, you mean "~~what~~ that"; I thought you were alluding to "spun up ~~to~~ from"